sip/sip-006-runtime-cost-assessment.md

```diff
@@ -136,6 +136,26 @@ where a and b are constants,
 X := the cumulative size of the argument types,
 costEval(body) := the cost of executing the body of the function
 
+### contract-call Transactions
+
+User-signed transactions for contract-calls are charged for the
+application of the function, as well as the loading of the contract
+data. This charge is the same as a normal contract-call. _However_,
+contract principals that are supplied as trait arguments must be
+checked by the runtime system to ensure that they validly implement
+the trait. The cost of this check is:
+
+```
+read_count = 2
+read_length = trait_size + contract_size
+runtime*cost = a*(contract_size) + b*(trait_size) + c
```

Comment:
Should `runtime*cost` be `runtime_cost`?

Reply:
Yep